### ISSUE TITLE
Docker image for 0.55.1 fails to start

### ISSUE THREAD

**Issue by sebk-666** [home-assistant/home-assistant.github.io#3646]
Hi,

I just updated my HA docker image from 0.55 to 0.55.1 and now it fails to start issuing the following error:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/src/app/homeassistant/__main__.py", line 374, in <module>
    sys.exit(main())
  File "/usr/src/app/homeassistant/__main__.py", line 349, in main
    args = get_arguments()
  File "/usr/src/app/homeassistant/__main__.py", line 90, in get_arguments
    import homeassistant.config as config_util
  File "/usr/src/app/homeassistant/config.py", line 26, in <module>
    from homeassistant.util.yaml import load_yaml, SECRET_YAML
  File "/usr/src/app/homeassistant/util/yaml.py", line 16, in <module>
    import credstash
  File "/usr/local/lib/python3.6/site-packages/credstash.py", line 55, in <module>
    'RIPEMD': hashes.RIPEMD160,
AttributeError: module 'cryptography.hazmat.primitives.hashes' has no attribute 'RIPEMD160'

```

Sebastian

**Comment by MartinHjelmare** [comment]
Duplicate of https://github.com/home-assistant/home-assistant/issues/9893.

Fixed in 0.55.2.

Btw, this is the home assistant docs repo issue tracker.

**Comment by sebk-666** [comment]
Sorry about mixing up the issue tracker URLs ;)
